heartbeat/Route

```diff
@@ -227,6 +227,15 @@ route_stop() {
 }
 
 route_status() {
+    if [ -n "${OCF_RESKEY_device}" ]; then
+	# Must check if device exists or is gone.
+	# If device is gone, route is also unconfigured.
+	device_exists="$(ip link show dev ${OCF_RESKEY_device})"
+	if [ ! $? -eq 0 ]; then
```

Comment:
You should use `-ne 0` instead of `! -eq`.

Reply:
Thanks, indeed that's more readable. Changed that and force-pushed :smile:.